src/app/modules/companies/store/companies.reducer.ts

```diff
@@ -0,0 +1,64 @@
+import { EditCompany } from './../../../models/edit-company.model';
+import { PartialDeep } from 'type-fest';
+import { createReducer, on } from '@ngrx/store';
+import * as CompaniesActions from './companies.actions';
+
+export const companiesFeatureKey = 'companies';
+
+export interface State {
+  selectedCompany: PartialDeep<EditCompany> | null,
+  editCompany: PartialDeep<EditCompany> | null,
```

Comment:
selectedCompany: PartialDeep<**EditCompany**> | null?

May be just Company then?

Reply:
Not fixed